Stylesheets/components/button.css

```diff
@@ -0,0 +1,294 @@
+@import url("/stylesheets/main.css");
+
+.btn-hover {
+  background-color: black;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
```

Comment:
text align is useless if you are using flex for the same container, so this can be removed

Reply:
okay .  got it.